<ui version="4.0" >
 <class>searchPACS</class>
 <widget class="QDialog" name="searchPACS" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PACS Query</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Search in...</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QRadioButton" name="patientName" >
          <property name="text" >
           <string>Patient Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="patientID" >
          <property name="text" >
           <string>Patient ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="patientBirthDate" >
          <property name="text" >
           <string>Patient Birth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="studyDate" >
          <property name="text" >
           <string>Study Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="accessionNumber" >
          <property name="text" >
           <string>Accession Number</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>No older than...</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QRadioButton" name="olderThanDay" >
          <property name="text" >
           <string>One day</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="olderThanMonth" >
          <property name="text" >
           <string>One month</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="olderThanWeek" >
          <property name="text" >
           <string>One week</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="olderThanEver" >
          <property name="text" >
           <string>No time limit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>searchPACS</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>searchPACS</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
